common/spec/dependabot/config/ignore_condition_spec.rb

```diff
@@ -86,7 +86,39 @@ def expect_ignored(versions)
         end
 
         it "returns the expected range" do
-          expect(ignored_versions).to eq(["> 1.2.3, < 1.3"])
+          expect(ignored_versions).to eq([">= 1.2.3.1.a, < 1.3"])
+        end
+
+        context "with a minor dependency version" do
+          let(:dependency_version) { "1.2" }
+
+          it "ignores expected updates" do
+            expect_ignored(["1.2.1.a", "1.2.1.1"])
+            expect_allowed(minor_upgrades + major_upgrades)
+            expect_allowed([dependency_version, "1.2.0"])
+          end
+        end
+
+        context "with a single major dependency version" do
+          let(:dependency_version) { "1" }
+          let(:patch_upgrades) { %w(1.0.0.1 1.0.2 1.0.5 1.0.4-rc0) }
+          let(:minor_upgrades) { %w(1.1 1.2.0 1.3 1.4.0) }
+
+          it "does not attempt to ignore any versions" do
```

Comment:
```suggestion
          it "ignores expected updates" do
```

This didn't seem right since we are testing that patches are ignored.